### PR TITLE
Add multifile to language's resource hash

### PR DESCRIPTION
**Pull request by felipecalvo**

### CODE REVIEW

File: app/models/language.rb [hunk 1]
```diff
@@ -67,7 +67,7 @@ def to_embedded_resource_h
   def to_resource_h
     as_json(only: %i(comment_type devicon editor_css_urls editor_html_urls editor_js_urls
                     extension feedback highlight_mode layout_css_urls layout_html_urls
-                    layout_js_urls name output_content_type prompt queriable runner_url
+                    layout_js_urls multifile name output_content_type prompt queriable runner_url
```
Comment (flbulgarelli) [inline app/models/language.rb]: We should add a migration, shoudn't we?

File: README.md [hunk 2]
```diff
@@ -17,6 +17,7 @@
   triable
   feedback
   queriable
+  multifile
```
Comment (flbulgarelli) [inline README.md]: :+1:

File: app/models/language.rb (hunk 1 above)
Comment (felipecalvo) [reply app/models/language.rb]: I've added this migration in a recent commit.

**Comment by flbulgarelli** [comment]
Related to https://github.com/mumuki/mumukit/issues/57

**Pull request merged**